packages/react/src/components/Link/Link.module.css

```diff
@@ -0,0 +1,28 @@
+.link,
+.link:visited {
```

Comment:
The :visited selector here looks the be wrong (since its also defined further down)?

Reply:
Good catch!